### DISCUSSION TITLE
FPS Drop on Embed

### DISCUSSION THREAD

**Discussion by stankikween** [cables-gl/cables_docs#200] [Answers & Questions] [answered]
Yo! I am a complete cables.gl newb and I am trying to embed a [patch](https://cables.gl/p/jOh2UY) I made into my Webflow website using iframe. The patch is pretty simple and by itself hovers around 60 fps. However, when I embed into my website, my fps drops to around 23. I have two videos that you can play on the website page but other than that it is very simple, and I'm not sure what is causing this drastic drop. Is there something common with embed optimization that I am missing? (My WIP website is[ here](https://molly-monahan.webflow.io/) if you want to check out the FPS drop)

Thanks so much!

**Comment by pandrr** [comment, marked as answer]
hi,

the fps already drops if you resize the renderer in cables

to optimize:

- turn off hdr for the render2texture
- reduce the amount of lines in linesarray
- turn off msaa in render2 texture and use fxaa, to get rid of aliasing you can combine fxaa with growpixels:

![Screenshot 2022-03-15 at 07 59 45](https://user-images.githubusercontent.com/98792/158323815-e7988f3a-3ee6-4aaa-819e-048fb9078d6b.png)

cheers!
